render50

```diff
@@ -198,12 +207,76 @@ def main():
     # Write documents to PDF
     # https://github.com/Kozea/WeasyPrint/issues/212#issuecomment-52408306
     if documents:
-        pages = [page for document in documents for page in document.pages]
-        documents[0].copy(pages).write_pdf(output)
+
+        # If we're in side-by-side mode, write two temp PDFs, then concatenate them
+        if args.side_by_side:
+
+            # Write temp PDFs for first and second document
+            documents[0].write_pdf('temp1.pdf')
+            documents[1].write_pdf('temp2.pdf')
+
+            merge_pdfs(output)
+
+            os.remove('temp1.pdf')
+            os.remove('temp2.pdf')
+        else:
+            pages = [page for document in documents for page in document.pages]
+            documents[0].copy(pages).write_pdf(output)
+
         cprint("Rendered {}.".format(output), "green")
     else:
         cprint("Nothing to render.", "red")
 
+def merge_pdfs(output):
+    """ 
+    Combines two temp PDFs page by page, side by side, filling in blank pages for shorter PDF side. 
+    https://stackoverflow.com/questions/24580454/python-pypdf2-join-pages/34646677#34646677
+    """
+    with open('temp1.pdf', 'rb') as temp1, open('temp2.pdf', 'rb') as temp2, open('blank.pdf', 'rb') as blank:
```

Comment:
Not appropriate to hardcode filenames here, breaks abstraction. Should be taken in as parameters.

Reply:
Do use double quotes for consistency.

Reply:
Hardcoded filenames are for the temp files generated before concatenation, not the input files :p

Reply:
Still messy, breaks the abstraction that is that method; everything should be cleanly encapsulated. Plus, if the student has files by the same name in their dir, we just clobbered them. Should instead use dynamically generated tempfiles in the system temp dir.

Reply:
Fair, though this is presumably fixed anyway by your suggestion to use `tempfile.mkstemp` above!